control.lua

```diff
@@ -45,11 +47,13 @@ script.on_init(function()
   end
 
   script.on_nth_tick(nth_tick, register_events)
+  script.on_event(defines.events.on_player_left_game, register_events)
```

Comment:
i prefer havin a function called register_events_players and add on_event for on_player_left_game / on_player_joined_game / on_player_kicked / on_player_removed

Reply:
on_player_banend too perhaps?

Reply:
Yeah this suggestion makes a lot more sense, should've switched to this when I realised I needed the on_palyer_left_game_hook

Reply:
does 353e8b1 cover it?